### DISCUSSION TITLE
Disable `X-Powered-By` Header by Default

### DISCUSSION THREAD

**Discussion by Maarten-Dekker** [serversideup/docker-php#417] [Ideas & Feature Requests]
I would like to propose disabling the `X-Powered-By` header by default. This header, which reveals the PHP version is currently included in HTTP responses. Sending these types of HTTP headers:
- does not provide any value to the user experience
- contributes to header bloat
- exposes information to potential attackers about the technology stack being used

See [https://webhint.io/docs/user-guide/hints/hint-no-disallowed-headers/?source=devtools](https://webhint.io/docs/user-guide/hints/hint-no-disallowed-headers/?source=devtools).

# Proposed Solution

Set `expose_php = On` to `expose_php = Off` by default in `php.ini`.

**Comment by jaydrogers** [comment]
Thanks for pointing this out! I will put this in the next release 👍
